v22.1/glossary.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
I think this glossary is great!

In terms of TOC, I might expect to find this lower down in this section by the FAQs, it feels a bit strange this high up in the tree.  Speaking for myself I'd be unlikely to read a glossary of terms before jumping into copypasting my sample app, etc.

if the idea is that users need this deployment info straightaway, I wonder if a good approach would be to add some links from the Quickstart page to the deployment options page.  (maybe that was already in another comment here? sorry for duplication if so)